### PR TITLE
replaces translateX with margins

### PR DESCRIPTION
**Pull request by gvonkoss**
The transform was leaving wiggle room in the browser as it doesn't look like things were aligning properly

### CODE REVIEW

File: src/scss/_themes.scss [hunk 1]
```diff
@@ -47,7 +47,10 @@
 		padding-left: 10px;
 
 		&:after {
-			left: 52%;
+			margin-left: calc(-50vw + 5px);
```
Comment (onishiweb) [inline src/scss/_themes.scss]: Can you explain where the `+5px` and `+10px` come from in a comment

File: src/scss/_themes.scss [hunk 2]
```diff
@@ -63,7 +66,7 @@
 				padding: 0 10px;
 			};
 
-			&:nth-child(4) {
+			&:last-child {
```
Comment (onishiweb) [inline src/scss/_themes.scss]: Why this change?

Comment (gvonkoss) [inline src/scss/_themes.scss]: modularity? What if it is suddenly decided that there are not 4 elements but 3?

Comment (onishiweb) [inline src/scss/_themes.scss]: Ah, but I thought that was designed to have rows of `4` always. And if we change it to rows of `3` etc. this component would change?

Comment (gvonkoss) [inline src/scss/_themes.scss]: This is true. I guess if you had never seen the output of the code it would make more sense to read last rather than fourth?

Comment (onishiweb) [inline src/scss/_themes.scss]: True, then it should be commented. Is there a case where this will go onto 2 rows? Or is it always going to be a single row. If the latter, `last-child` is fine

Comment (gvonkoss) [inline src/scss/_themes.scss]: Each teaser breaks into its individual row below `M`...

Comment (gvonkoss) [inline src/scss/_themes.scss]: Or are you referring to a comment as to _why_ I made a change, not what that bit is doing?
